src/Umbraco.Core/Models/Mapping/MemberTabsAndPropertiesMapper.cs

```diff
@@ -46,7 +54,32 @@ public class MemberTabsAndPropertiesMapper : TabsAndPropertiesMapper<IMember>
         _memberService = memberService ?? throw new ArgumentNullException(nameof(memberService));
         _memberGroupService = memberGroupService ?? throw new ArgumentNullException(nameof(memberGroupService));
         _memberPasswordConfiguration = memberPasswordConfiguration.Value;
-        _propertyEditorCollection = propertyEditorCollection;
+        _twoFactorLoginService = twoFactorLoginService ?? throw new ArgumentNullException(nameof(twoFactorLoginService));
+    }
+
+    [Obsolete("Use other ctor - Will be removed in Umbraco 13")]
```

Comment:
We should explain what the other constructor is to be helpful 😁 
So should be `Use constructor that also takes a ITwoFactorLoginService, this is scheduled for removal in V13 `

Reply:
I will be more concrete then 👍